backend/requirements_worker.txt

```diff
@@ -13,7 +13,8 @@ pydantic==1.9.1
 pydantic[email]==1.9.1
 python-dotenv==0.21.0
 python-jose[cryptography]==3.3.0
-redis
+redis==4.5.5
+requests==2.30.0
```

Comment:
As above, are these requirement versions needed for the new functionality to work?

Reply:
Those are the latest version of the requirements, I added them to have the requirements version pinned in the file.